.chloggen/sqlserver-top-query-events.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sqlserverreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use generated structured event functions in mdatagen
```

Comment:
This is also end-user facing change since they now are able to disable particular evetns. Can be another changelog item I guess

Reply:
But give that it's only one event, it's not very useful. So we probably don't need it